packages/base/card-api.gts

```diff
@@ -755,7 +757,11 @@ class LinksTo<CardT extends CardConstructor> implements Field<CardT> {
       return {
         relationships: {
           [this.name]: {
-            links: { self: value.reference },
+            links: {
+              self: opts?.maybeRelativeURL
```

Comment:
Is there a way to avoid the repetition of `opts?.maybeRelativeURL ? opts.maybeRelativeURL(val) : val`? Perhaps by assigning a `normalizeURL` or `formatURL` function once and then using it in all these places?

Reply:
This is the function that I'm talking about: https://github.com/cardstack/boxel/blob/0376a5e61c8003ea66feff69140caf89b8484f21/packages/base/card-api.gts#L1841-L1862. The idea is maintain the `modelRelativeTo` as we recurse thru all the fields in the serialize function.